ImperatorToCK3/Source/Mappers/NicknameMapper/NicknameMapping.cpp

```diff
@@ -0,0 +1,18 @@
+#include "NicknameMapping.h"
+#include "ParserHelpers.h"
+
+mappers::NicknameMapping::NicknameMapping(std::istream& theStream)
+{
+	registerKeyword("ck3", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
This is perfectly legit but stupid IDE marks theStream as shadowing theStream from outer function (which is of course true) - which is why I started moving registerKeywords into registerKeys() function in every parser class. Purely cosmetic and to eliminate the blue underlines in VS.